metecho/api/serializers.py

```diff
@@ -48,6 +49,48 @@ def to_representation(self, value):
         return str(value.pk)
 
 
+class GitHubIssueSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(read_only=True)
+    project = HashidPrimaryKeyRelatedField(queryset=Project.objects.all())
+    epic = serializers.SerializerMethodField()
+    task = serializers.SerializerMethodField()
+
+    class Meta:
+        model = GitHubIssue
+        read_only_fields = (
+            "id",
+            "number",
+            "title",
+            "created_at",
+            "html_url",
+            "project",
+            "epic",
+            "task",
+        )
+        fields = read_only_fields
+
+    def get_epic(self, issue):
+        try:
+            return {
+                "id": str(issue.epic.id),
+                "name": issue.epic.name,
+                "status": issue.epic.status,
+            }
```

Comment:
To generate the link to the Epic Detail page, I think we'll also need the `issue.epic.slug`.

Reply:
Fixed in 94e344587db6a311467b76c6be3f1831c8a2c96f